applications/connecting_applications_to_services/exposing-binding-data-from-a-service.adoc

```diff
@@ -21,6 +21,7 @@ include::modules/sbo-categories-of-exposable-binding-data.adoc[leveloffset=+1]
 [role="_additional-resources"]
 [id="additional-resources_exposing-binding-data"]
 == Additional resources
-* link:https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md[OLM Descriptor Reference].
+// * link:https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md[OLM Descriptor Reference].
+// Add this additional resource once OLM descriptors are supported again.
```

Comment:
IBM SG: use `when` or `after` instead of `once. 

`Add this additional resource after/when OLM descriptors are supported again.`
or
`When OLM descriptors are supported again, add this additional resource.`